picard/util/webbrowser2.py

```diff
@@ -33,15 +34,13 @@
 
 from PyQt6 import QtWidgets
 
-from picard.const import PICARD_URLS
 from picard.i18n import gettext as _
+from picard.util.readthedocs import ReadTheDocs
 
 
 def open(url):
-    if url in PICARD_URLS:
-        url = PICARD_URLS[url]
     try:
-        webbrowser.open(url)
+        webbrowser.open(ReadTheDocs.get_url(url))
```

Comment:
Not a big thing, but I find it a bit odd that `get_url` is defined in `ReadTheDocs`, as it is actually not specific to only documentation links. Maybe we should just put this into `utils`.